cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
+
+Once your new private space is ready, you can gradually migrate your applications from the CloudHub VPC to the CloudHub 2.0 private space. After all your applications are successfully moved, you can decommission your old VPC.
+
+
+// Benefits
+Using this VPC in-product upgrade tool saves you time by eliminating the need to acquire new CIDR blocks, set up new network connections, and configure the private space from the ground up.
+
+The tool provisions the infrastructure for a CloudHub 2.0 private space by cloning the existing configurations of your eligible CloudHub dedicated VPC, using the same CIDR block. 
+
+If your VPC has existing Transit Gateways (TGW) or Virtual Private Networks (VPN) connections, they are seamlessly transferred to the new CloudHub 2.0 private space. This ensures that any new applications deployed into the new private space can communicate effectively.
+
+The default Cloudhub certificates are configured as part of the private space's default certificates. The DLB custom domains of the migrated VPC are created as part of the private space's custom TLS certificates.
+
+[NOTE]
+During the VPC upgrade process, you use doubled entitlements for both VPC & Network Connections. Work with your account representative to make sure you stay compliant on entitlements and finish the upgrade on time.
+
+== Understand Eligibility
+//Which VPCs are eligible for upgrade
+
+Due to their architectural differences and distinct interfaces, CloudHub 2.0 doesn't support some features in the same way CloudHub does. Follow this eligibility checklist to confirm which VPCs are eligible for the upgrade:
+
+Availability of IPs::
++
+Your VPC must have at least 25% free space, and 100 or more available IP addresses per subnet to be eligible for the upgrade.
+
+Deprecated Availability Zones (AZs)::
++
+VPCs using deprecated AZs aren't eligible for the upgrade.
+
+Legacy VPNs::
++
+VPCs with legacy VPNs configured aren't eligible for the upgrade.
+
+Direct Connect or VPC Peering:: 
+If your VPC has Direct Connect or VPC Peering configured, migrate to Transit Gateway or VPN before upgrading.
+
+TLS 1.1 ciphers::
++
+These ciphers can't be carried over from CloudHub to CloudHub 2.0:
++
+* ECDHE-ECDSA-AES128-SHA1
+* ECDHE-ECDSA-AES256-SHA1
+* ECDHE-RSA-AES128-SHA1
+* ECDHE-RSA-AES256-SHA1
+
+
+== Upgrade your VPC via Anypoint Platform
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Upgrade*.
++
+Alternatively, select *Runtime Manager* > *What's New* to read more about the VPC upgrade, and click *Continue to Upgrade* to start upgrading your VPCs.
+. Click *Upgrade Now*, under the *Upgrade Status* column for the VPC you want to upgrade.
++
+The *Upgrade to CloudHub 2.0* review page displays the configurations currently available in your VPC, as well as the configurations that are created during the upgrade.
++
+image::upgrade-ch2-review-conf.png[]
```

Comment:
Can you add alt text here? ["Upgrade to CloudHub 2.0 review page showing available configurations"]
(or whatever the image is...)